lib/matplotlib/widgets.py

```diff
@@ -364,8 +364,9 @@ def __init__(self, ax, label, valmin, valmax, *, valinit=0.5, valfmt=None,
             The slider initial position.
 
         valfmt : str, default: None
-            %-format string used to format the slider value.  If None, a
-            `.ScalarFormatter` is used instead.
+            %-format string used to format the slider value. If callable, will
+            be called with the value as a single float argument, and should
+            return a string. If None, a `.ScalarFormatter` is used instead.
```

Comment:
I think this wording as a bit more concise:
```suggestion
            The way to format the slider value. If a string, if must be in %-format.
            If a callable, it must have the signature ``valfmt(val: float) -> str``.
            If None, a `.ScalarFormatter` is used.
```